src/Symfony/Component/Cache/CacheInterface.php

```diff
@@ -30,8 +30,11 @@
 {
     /**
      * @param callable(CacheItemInterface $item):mixed $callback Should return the computed value for the given key/item
+     * @param float                                    $beta     A float that controls the likeness of triggering early expiration.
+     *                                                           0 disables it, INF forces immediate expiration. The default is implementation dependent
+     *                                                           but should typically be 1.0, which should provide optimal stampede protection.
      *
      * @return mixed The value corresponding to the provided key
      */
-    public function get(string $key, callable $callback);
+    public function get(string $key, callable $callback, float $beta = null);
```

Comment:
I don't like unclear default values. Why not `1.0` directly? Also it's not explained what null means in the phpdoc.

Reply:
It's also not like this interface has different implementations with different default values.